### ISSUE TITLE
WebSockets guide hard to follow

### ISSUE THREAD

**Issue by Radiergummi** [spiral/docs#189]
Adding websockets to an application by following the documentation had some pitfalls I'd like others to avoid:

> The default GRPC and Web bundles include a pre-build component to gain access to the broadcast topic via WebSocket client. To activate the component, enable the `ws` section `.rr.yaml` config file with the desired path:
>
> ```yaml
> ws.path: "/ws"
> ```
This leaves me wondering - what other options for the WS components aside from `path` are there? There is no documentation for it, neither in the Spiral nor the Roadrunner docs.

----

> **Usage sample to listen a specific channel**  
> ```js
> import { SFSocket } from '@spiralscout/websockets';
>
> const socketOptions = { host: 'localhost' };
>
> const ws = new SFSocket(socketOptions);
>
> SFSocket.ready();
> 
> // ...
> 
> // disconnect everything
> ws.disconnect()
> ```

Using the code sample for the spiral websocket client, the connection fails due to an unexpected end of file error on the server:
```
DEBU[0009] [ws] 172.21.0.1:62314 websocket: close 1006 (abnormal closure): unexpected EOF 
```
This, of course, is because the connection is immediately terminated to showcase closing connections, but I would suggest commenting it out or change the sample to something that works out of the box. The error is obvious once you know what it is caused by, but confusing if you're getting started.

----

> **Consume Topic**  
> To consume topic using JavaScript client your must send command message in a form of {"topic":"join", "payload": ["topic-name"]}. Modify your script to automatically consume topic:
> ```js
> // ...
> ```
If the recommended way of connecting is the Spiral websockets client, why do the following code samples continue the native example? I mean, it's not hard to rewrite the code to use the client, but it would be nice if the docs were consistent here.

**Comment by butschster** [comment]
Hello @Radiergummi 

Sorry for long response! I've just finished writing docs for websockets. Here  https://spiral.dev/docs/websockets-services/3.3/en you will find almost everything you need about using websockets in Spiral Framework.
